package.json

```diff
@@ -35,6 +35,7 @@
     "start": "nebula serve --type sn-org-chart",
     "test:unit": "aw -c aw.config.js",
     "test:integration": "aw puppet --glob 'test/integration/**/*.spec.js' --chrome.headless true --chrome.slowMo 10",
+    "test:rendering": "aw puppet -c aw.ren.config.js --testExt '*.spec.js' --glob 'test/rendering/**/*.spec.js'",
```

Comment:
Why are we still using after work for this? Can't we fix both of them at the same time?

Reply:
Will be a new test-assistant in @nebula.js/test-utils to replace after work which sets up the puppeteer and does other repetitive work. but that is not ready yet.